### DISCUSSION TITLE
No connection

### DISCUSSION THREAD

**Discussion by Leoneed67** [apohl79/audiogridder#344] [Help]
Hi! The first version worked, and the latest updates do not allow you to join the server! The server I have installed on a computer with Windows 10. The plugin on the Mac does not connect to the server! The computers are connected via wifi.

**Comment by Leoneed67** [comment]
One more thing! In the first version, I assigned the server an IP and the connection occurred without problems! Now I am trying to assign an IP, but when saving this does not happen, only the first digits are saved. An attempt to contact the server by name or by these stored digits fails!

**Comment by xcorio** [comment]
Try to reset your audiogridder installation by wiping all config files: https://audiogridder.com/bug-report-diagnostics-locations/

**Reply by Leoneed67** [reply]
I did everything, but it didn't help!

**Comment by CaBuffalo** [comment]
It looks like a network issue. Check the firewall on your system to see if
there is any block to port 55054 / 55055 / 55056.

"Now I am trying to assign an IP, but when saving this does not happen,
only the first digits are saved." -- do you mean Windows 10 won't save the
IP address you assigned manually?

On Sun, Jan 31, 2021 at 4:03 PM Leoneed67 <notifications@github.com> wrote:

> I did everything, but it didn't help!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/344#discussioncomment-326066>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CQYPWOK5RIRWRHWT3S4XVTZANCNFSM4W274NQA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by tanguytree** [comment]


**Reply by tanguytree** [reply]
<img width="1429" alt="Capture d’écran 2021-02-13 à 17 54 45" src="https://user-images.githubusercontent.com/78609868/107855795-c355f300-6e24-11eb-8a0d-a9f0c276016e.png">

**Comment by CaBuffalo** [comment]
You need to make sure both of your system s are in the same LAN.

Can you post your IP configuration of both of your systems?

On Sat, Feb 13, 2021 at 8:57 AM tanguytree <notifications@github.com> wrote:

> i linked both by RJ45 thunderbolt. It worked before
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/344#discussioncomment-366230>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DYCOUAO5JFRIWTSLTS62VOZANCNFSM4W274NQA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by tanguytree** [reply]
<img width="414" alt="Screen Shot 2021-02-19 at 12 03 59" src="https://user-images.githubusercontent.com/78609868/108496611-95e8c980-72aa-11eb-903f-93a78764b0ec.png">
server

**Reply by tanguytree** [reply]
![IMG_20210219_120610](https://user-images.githubusercontent.com/78609868/108497126-4951be00-72ab-11eb-8a82-f39495473f86.jpg)
plugin

**Comment by CaBuffalo** [comment]
Your server didn't get the correct IP for some reason. It looks like it's
not connect to the DHCP server / router.

Your DAW and server machines are not in the same network.

On Fri, Feb 19, 2021 at 3:09 AM tanguytree <notifications@github.com> wrote:

> [image: IMG_20210219_120610]
> <https://user-images.githubusercontent.com/78609868/108497126-4951be00-72ab-11eb-8a82-f39495473f86.jpg>
> plugin
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/344#discussioncomment-384215>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CVTFTHJIU7OL4YQNDS7ZBH5ANCNFSM4W274NQA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Gilligan8** [reply]
I know this is super old but in case anyone is looking at this for help.

They are on the same network, this is the fall back APIPA address (that address part is redundant) this is so when computers can't find a DHCP server they still fall back to an address where they can all talk.

We are used to seeing Class C networks 192.168.1.X with a subnet of 255.255.255.0  That last subnet part is the important distinction you see here.

The APIPA network has a B Class subnet so it can hold 65,536 host vs 252 on the Class C.

The 169.254.x.x is the only important octets to make sure these computers are on the same network.
The 255.255.0.0 as you see it lining up with those two 255's is why it's important.  They can't be even one number off or it won't work  But the Zeros mean it can be ANY of the 0-255's (aside from some reserves) and it's all on the same network.

Any how, back to my reading to sort out my connection issue. :)

**Comment by looshamik** [comment]
I don't know if anyone will see this, having spent almost three years but, which of the data shown in the screenshots do you look at to see that the machines are not on the same network?. How would you solve this?. In my case, I am trying to configure two macs running macOS Monterey, and I am encountering a situation similar to the one exposed at the beginning of this conversation.SER
![Captura de Pantalla 2023-12-18 a las 14 19 25](https://github.com/apohl79/audiogridder/assets/115983188/ce5e0ad6-c448-4440-a38e-69556f0f9335)
![Captura de Pantalla 2023-12-18 a las 14 21 21](https://github.com/apohl79/audiogridder/assets/115983188/8af0d3e0-ae66-4a61-9d34-cb7728a3d865)
Thanks!

**Reply by Gilligan8** [reply]
These are not on the same network... see my comment above to understand how the subnetting is the important part of figuring that out.

I also realize this is 2 years after... never too late to learn. :)

**Comment by ThaSequel1280** [comment]
My audiogridder is not conncting to a server

**Reply by looshamik** [reply]
Hello. What kind of connection do you have between the main computer and the server? I have been running AudioGrider until recently and practically without problems connecting my macs via thunderbolt bridge. There were connection problems when I had a VPN installed, but if I uninstalled it the problems disappeared.

**Reply by looshamik** [reply]
Apart from the VPN issue, when configuring the
network (thunderbolt bridge) you have to configure IPv4 by selecting manual mode and assigning an IP address specifically defined for the thunderbolt bridge. In my case, for my client (main) computer:
![Captura de Pantalla 2025-01-29 a las 13 43 17](https://github.com/user-attachments/assets/5365653b-af7c-42e2-af59-40d7464cadca)
On the server it is the same but with the IP address: 13.13.13.13
I don't know if this will help you.
Regards
